.github/workflows/update-k8s-versions.yml

```diff
@@ -1,8 +1,8 @@
 name: "update-kubernetes-versions"
 on:
   schedule:
-    # every week on Thursday at 20:30 UTC
-    - cron: "45 20 * * 4"
+    # every day at around 1 am pacific/8 am UTC
+    - cron: "0 8 * * *"
```

Comment:
everyday is not good, incase we want to skip a release, it will keep bugging us every day...(so it is better it bugs us every week)

Reply:
fixed